<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateElderDialog</class>
 <widget class="QDialog" name="UpdateElderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="updateFirstNameLabel">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="newFirstNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="updateMiddleNameLabel">
       <property name="text">
        <string>Middle Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="newMiddleNameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="updateLastNameLabel">
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="newLastNameLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="updatePhoneNumberLabel">
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="newPhoneNumberLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="updateCongregationLabel">
       <property name="text">
        <string>Congregation:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="newCongregationComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="updateTalkNumberLabel">
       <property name="text">
        <string>Talk Number:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="newTalkNumberComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="newEnabledStatusLabel">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="newEnabledStatusCheckbox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateElderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateElderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
